exec/pipeline/financials/structs.py

```diff
@@ -0,0 +1,128 @@
+from pydantic import BaseModel, Field
+from decimal import Decimal
+
+
+class NetSales(BaseModel):
+    products: Decimal
+    services: Decimal
+    total_net_sales: Decimal
+
+class CostofSales(BaseModel):
+    products: Decimal
+    services: Decimal
+    total_cost_of_sales: Decimal
+
+
+
+class OperatingExpenses(BaseModel):
+    research_and_development: Decimal
+    selling_general_and_administrative: Decimal
+    total__operating_expenses: Decimal
```

Comment:
Just noticed it looks like there's two underscores not sure if that's intentional.

Reply:
good catch, fixed